antenna/ext/crashstorage_base.py

```diff
@@ -17,14 +17,14 @@ class CrashStorageBase(RequiredConfigMixin):
     def __init__(self, config):
         self.config = config.with_options(self)
 
-    def save_raw_crash(self, raw_crash, dumps, crash_id):
+    def save_raw_crash(self, crash_id, raw_crash, dumps):
```

Comment:
On the call, I was thinking "seems better to save everything in one pass rather than split it up", but now I'm thinking that's a lot of extra work if you're just making changes to the raw crash metadata and want to save that.

I'm persuaded. I'll split it up into `save_raw_crash` and something else. Maybe `save_dumps` or maybe I'll just do `save_dump` and move the iteration to the parent.